content/en/docs/setup/platform-setup/prerequisites/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Platform Prerequisites
+description: Prerequisites for platform setup for istio.
+weight: 1
+skip_seealso: true
+aliases:
+    - /docs/setup/kubernetes/prepare/platform-setup/prerequisites/
+    - /docs/setup/kubernetes/platform-setup/prerequisites/
+keywords: [platform-setup,prerequisites]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+
+## Kernel Module Requirements on Cluster Nodes
+
+The following kernel modules must be loaded on the cluster node running application pods with a Service Mesh proxy sidecar container.
+The modules are needed specifically by the `istio-init` container which sets up iptables rules in the pod to redirect any incoming or
+outgoing traffic towards the sidecar proxy in the istio-proxy container. While in many platforms, these seem to be automatically
+loaded, it is always good to make sure the prerequisites are met, as there were incidents reported where some of the specific
+modules listed down below were not available on the host or could not be automatically loaded by the iptables. For example,
+[`selinux issue`](https://www.suse.com/support/kb/doc/?id=000020241) talks about selinux in RHEL sometimes preventing
+the automatic loading of some of the below mentioned kernel modules.
+
+| Module | Remark |
+| --- | --- |
+| `br_netfilter` |  |
+| `ip6table_mangle` | Only needed for IPv6 or Dual-stack clusters |
```

Comment:
I'm not sure if dual stack or dual-stack is the correct way to write this. Either way, I think it should not be capitalized in the tables.